ledger/heavy/executor/jetkeeper.go

```diff
@@ -262,15 +330,13 @@ func (jk *dbJetKeeper) updateDrop(ctx context.Context, pulse insolar.PulseNumber
 	return jk.set(pulse, jets)
 }
 
-func infoToSet(s []jetInfo) (map[insolar.JetID]struct{}, bool) {
+func infoToSet(s []jetInfo, checkBackup bool) (map[insolar.JetID]struct{}, bool) {
```

Comment:
something wrong in using this function for a new purpose, need a rewrite

Reply:
"smith wrong" - could you clarify it?